surelock/src/main/java/com/smashingboxes/surelock/SurelockDefaultDialog.java

```diff
@@ -38,8 +39,8 @@
     private static final long ERROR_TIMEOUT_MILLIS = 1600;
     private static final long SUCCESS_DELAY_MILLIS = 1300; //TODO make these configurable via attrs
 
-    private FingerprintManager fingerprintManager;
```

Comment:
Do we need to also make this class extend `AppCompatDialogFragment`?
I'm not sure if it's necessary